packages/connect-query/src/unary-hooks.ts

```diff
@@ -208,8 +208,13 @@ export const unaryHooks = <I extends Message<I>, O extends Message<O>>({
 
       ...(getPlaceholderData
         ? {
-            placeholderData: () =>
-              new methodInfo.O(getPlaceholderData(enabled)),
+            placeholderData: () => {
+              const placeholderData = getPlaceholderData(enabled);
+              if (placeholderData === undefined) {
+                return undefined;
```

Comment:
thought: I was surprised to find, then, that https://eslint.org/docs/latest/rules/consistent-return isn't on.  oh well. that's fine I guess.

```suggestion
                return;
```